Version 1.17.8

matteobruni opened the pull request:
fix: fixes #700

matteobruni committed:
fix: fixes #700

ghost commented:
## DeepCode's analysis on [#430f86](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/4123c367a0430aa5bfc2ad238406625dff032b38/matteobruni/tsparticles/430f8607a691e69341d4b41c58d739ecf8cd9430/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) found:

- :information_source: **1** minor issue. :point_down:
- :heavy_check_mark: **1** issue was fixed.
## Top issues

<table>
  <thead>
    <tr>
      <th align="left">Description</th>
      <th align="left">Example fixes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td width="77%">Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/430f8607a691e69341d4b41c58d739ecf8cd9430/components/inferno/src/ParticlesProps.ts#L15">ParticlesProps.ts:15</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/430f8607a691e69341d4b41c58d739ecf8cd9430/components/react/src/ParticlesProps.ts#L15">ParticlesProps.ts:15</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/430f8607a691e69341d4b41c58d739ecf8cd9430/components/preact/src/ParticlesProps.ts#L15">ParticlesProps.ts:15</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/4123c367a0430aa5bfc2ad238406625dff032b38/matteobruni/tsparticles/430f8607a691e69341d4b41c58d739ecf8cd9430/pr/_/%2Fcomponents%2Finferno%2Fsrc%2FParticlesProps.ts/javascript%2Ftslint%2Fno-any/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
  </tbody>
</table>


#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/4123c367a0430aa5bfc2ad238406625dff032b38/matteobruni/tsparticles/430f8607a691e69341d4b41c58d739ecf8cd9430/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)

matteobruni committed:
chore(release): published new version

 - ng-particles@2.0.8
 - inferno-particles@1.0.7
 - jquery-particles@1.17.8
 - preact-particles@1.17.8
 - react-tsparticles@1.17.8
 - svelte-particles@1.0.8
 - particles.vue@2.0.8
 - tsparticles-editor@1.0.0-1.0.0-alpha.37.1
 - tsparticles@1.17.8
 - ng-particles-demo@1.2.8
 - editor-demo@1.0.0-1.0.0-alpha.38.1
 - inferno-particles-demo@1.0.7
 - jquery-particles-demo@1.2.8
 - tsparticles-demo@1.2.8
 - preact-particles-demo@1.2.8
 - react-tsparticles-demo@1.2.8
 - svelte-demo@1.0.8
 - particles.vue-demo@1.2.8
 - tsparticles-preset-60fps@1.2.8
 - tsparticles-preset-background-mask@1.2.8
 - tsparticles-preset-basic@1.2.8
 - tsparticles-preset-bouncing@1.2.8
 - tsparticles-preset-fire@1.2.8
 - tsparticles-preset-font-awesome@1.2.8
 - tsparticles-preset-snow@1.2.8
 - tsparticles-preset-stars@1.2.8
 - tsparticles-shape-bubble@1.2.8
 - tsparticles-shape-heart@1.2.8
 - tsparticles-shape-multiline-text@1.2.8
 - tsparticles-shape-spiral@1.2.8
 - cra-template-particles-typescript@1.0.8
 - cra-template-particles@1.0.8
